chia/util/config.py

```diff
@@ -48,10 +48,21 @@ def config_path_for_filename(root_path: Path, filename: Union[str, Path]) -> Pat
     return root_path / "config" / filename
 
 
-def get_config_lock(root_path: Path, filename: Union[str, Path]) -> BaseFileLock:
+@contextlib.contextmanager
+def lock_config(root_path: Path, filename: Union[str, Path]) -> Iterator[None]:
```

Comment:
I believe the rename here was kind of a refactor aid to make sure I visited all usages.  I think I might like it better, but I can change it back as well.

Reply:
+1 for the `lock_config` name